src/fides/api/ops/service/messaging/message_dispatch_service.py

```diff
@@ -456,18 +457,32 @@ def _twilio_email_dispatcher(
         )
 
     try:
+
         sg = sendgrid.SendGridAPIClient(
             api_key=messaging_config.secrets[
                 MessagingServiceSecrets.TWILIO_API_KEY.value
             ]
         )
+
+        # the pagination via the client actually doesn't work
+        # in lieu of over-engineering this we can manually call
+        # the next page if/when we hit the limit here
+        response = sg.client.templates.get(
+            query_params={"generations": "dynamic", "page_size": 200}
```

Comment:
seems fine to me to hardcode for now to get this across the line, per your comment in the description. a couple of notes though:

1. i'd definitely create a follow up issue to track the improvement just so it doesn't go completely forgotten/untracked
2. i'd put the hardcoded limit in a docs update (see general comment for making sure we update our docs) to make the current limitation clear to anyone using it, just to reduce the amount of digging we have to do if/when we run into the limitation
3. any reason you've chosen `200` here as the hardcoded page size (and effective limit)? it feels like it could be a bit higher here and we wouldn't get overwhelmed with data, while just reducing the likelihood that we're impacted by the hardcoding. but that's just based on an uninformed gut feeling, so please ignore me if i'm spouting nonsense!

Reply:
Thanks again @adamsachs !!

1. Docs ticket still needed, will create here now and start working.
2. Great idea, I probably wouldn't have done that 🙌🏽 
3. The hard-coded limit of `200` comes from the Sendgrid API itself (I put 1000 and the response conveniently lists that max as 200)

For the end to end test, I would love some help here as I hit a snag following the docs and was about to write a bug issue. Will reach out in Slack

Reply:
👍 nice, thanks for the follow up and clarification. sendgrid API 🙄